storage/container_storage_interface/persistent-storage-csi-ebs.adoc

```diff
@@ -0,0 +1,30 @@
+[id="persistent-storage-csi-ebs"]
+= AWS Elastic Block Store CSI Operator
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-ebs
+
+toc::[]
+
+== Overview
+
+Storage vendors have traditionally provided storage drivers as part of Kubernetes. With the implementation of the Container Storage Interface (CSI), third-party providers can instead deliver storage plug-ins using a standard interface without ever having to change the core Kubernetes code.
+
+CSI Operators give {product-title} users storage options that are not possible with in-tree volume plug-ins, such as volume snapshots and cloning.
+
```

Comment:
We did not support cloning in OCP4.5

Reply:
> We did not support cloning in OCP4.5

We do support CSI cloning as of [4.4](https://docs.openshift.com/container-platform/4.4/storage/container_storage_interface/persistent-storage-csi-cloning.html), and it's moving from TP to GA in [4.5](https://issues.redhat.com/browse/STOR-311).

Reply:
@bobfuru Yeah, we moved CSI clone feature from TP to GA in 4.5.  But this only means the feature in the csi controller side, for AWS EBS CSI driver, it did not implement clone feature, so when the end-user uses AWS EBS CSI driver, he/she can not create a clone successfully.

Reply:
> @bobfuru Yeah, we moved CSI clone feature from TP to GA in 4.5. But this only means the feature in the csi controller side, for AWS EBS CSI driver, it did not implement clone feature, so when the end-user uses AWS EBS CSI driver, he/she can not create a clone successfully.

OK, I see now the discussion in https://issues.redhat.com/browse/STOR-344. I thought that CSI cloning was supported by this Operator based on the third bullet in https://github.com/openshift/enhancements/blob/master/enhancements/storage/csi-ebs-operator.md#motivation.

I've removed "and cloning" here.